test/parallel/test-crypto-fips.js

```diff
@@ -89,23 +107,23 @@ testHelper(
 testHelper(
   'stdout',
   [`--openssl-config=${CNF_FIPS_ON}`],
-  compiledWithFips() ? FIPS_ENABLED : FIPS_DISABLED,
+  FIPS_ENABLED,
```

Comment:
In this case of a fips-less build OpenSSL with throw an error which is [handled](https://github.com/nodejs/node/pull/13786/commits/bb6a1a2574e56f08956461b790b7ab9aa4dc54fd#diff-777496213f728df7a098c6eaf9ff482bR64) and the asserts skipped. Previously this error went undetected an the process returned 0, but when the same test was run with a build having an OpenSSL FIPS compatible version the call will succeed and return 1.